src/main/java/com/faforever/client/query/SearchableProperties.java

```diff
@@ -12,34 +12,31 @@
  */
 public class SearchableProperties {
   public static final Map<String, String> GAME_PROPERTIES = ImmutableMap.<String, String>builder()
+      .put("playerStats.player.login", "game.player.username")
+      .put("playerStats.player.globalRating.rating", "game.player.globalRating")
+      .put("playerStats.player.ladder1v1Rating.rating", "game.player.ladderRating")
+      .put("featuredMod.technicalName", "featuredMod.technicalName")
+      .put("mapVersion.map.displayName", "game.map.displayName")
+      .put("playerStats.faction", "game.player.faction")
+      .put("playerStats.startSpot", "game.player.startSpot")
+      .put("mapVersion.maxPlayers", "map.maxPlayers")
+      .put("mapVersion.ranked", "game.map.isRanked")
       .put("id", "game.id")
```

Comment:
Are you sure we wanna put id that far down?

Reply:
I asked @PhilipJFryFAF for an order. And yes almost no user knows game ids